test-utils/src/dev/db.rs

```diff
@@ -1101,11 +1152,16 @@ mod test {
         let extra_temp_dir =
             tempdir().expect("failed to create temporary directory");
         let data_dir = extra_temp_dir.path().join("custom_data");
-        let starter = new_builder().store_dir(&data_dir).build().unwrap();
+        let mut builder = new_builder().store_dir(&data_dir);
+        builder.env("GOTRACEBACK", "bogus").env("OMICRON_DUMMY", "dummy");
+        let starter = builder.build().unwrap();
 
         // This common function will verify that the entire temporary directory
         // is cleaned up.  We do not need to check that again here.
-        test_setup_database(starter, &data_dir, false).await;
+        let mut env_overrides = BTreeMap::new();
+        env_overrides.insert("GOTRACEBACK", "bogus");
+        env_overrides.insert("OMICRON_DUMMY", "dummy");
```

Comment:
Nitpick, can we declare this first, and use it to call `builder.env`? Just to "set the values once"

Reply:
Good call.

Reply:
Fixed in e71b56b2.